func@0.18.rb

```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class Func < Formula
```

Comment:
```suggestion
class FuncAT018 < Formula
```

You need to rename the ruby class to something unique (see also the other, version specific fomulas)